primefaces/src/main/resources/META-INF/resources/primefaces/accordion/accordion.js

```diff
@@ -100,16 +100,21 @@ PrimeFaces.widget.AccordionPanel = PrimeFaces.widget.BaseWidget.extend({
                 element.removeClass('ui-state-hover');
             }
         }).on("click", function(e) {
-            var element = $(this);
-            if(!element.hasClass('ui-state-disabled')) {
+            var header = $(this);
+            if(!header.hasClass('ui-state-disabled')) {
                 var tabIndex = $this.headers.index(element);
 
-                if(element.hasClass('ui-state-active')) {
+                if(header.hasClass('ui-state-active')) {
                     $this.unselect(tabIndex);
                 }
                 else {
                     $this.select(tabIndex);
-                    $(this).trigger('focus.accordion');
+                    header.trigger('focus.accordion');
+                    this.scrollIntoView({
```

Comment:
even better because I have set "center" "center" it scrolls the value into the center of the viewport not the top or bottom of viewport.  To me that is expected behavior and it works great.